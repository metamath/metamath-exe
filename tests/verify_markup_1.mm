$c c $. $v v $.
vv $f c v $.
ok $p c v $= ? $.
$( $t
  latexdef 'c' as 'c';
  htmldef 'c' as 'c';
  althtmldef 'c' as 'c';
  latexdef 'v' as 'v';
  htmldef 'v' as 'v';
  althtmldef 'v' as 'v';
$)
