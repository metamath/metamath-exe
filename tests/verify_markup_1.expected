MM> READ "verify_markup_1.mm"
Reading source file "verify_markup_1.mm"... 196 bytes
196 bytes were read into the source buffer.
The source has 4 statements; 0 are $a and 1 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> Checking statement label conventions...
Checking latexdef, htmldef, althtmldef...
Checking statement comments...
?Warning: There is no "(Contributed by...)" in the comment above statement 4,
    label "ok".
Checking section header comments...
Checking bibliographic references...
Checking mathbox independence...
